packages/forms-web-app/src/controllers/projects/examination-timetable/utils/events/utils/fetch-events.js

```diff
@@ -0,0 +1,9 @@
+const { getTimetables } = require('../../../../../../services/timetable.service');
+
+const fetchEvents = async (caseRef) => {
+	const { data } = await getTimetables(caseRef);
+
+	return data?.timetables || [];
```

Comment:
the || [] should mean the ? is not needed ?

Reply:
have you tested and empty array returned ?

Reply:
@jonathangoulding I have added a test for this now. The api can return undefined which will break the function if the optional chaining is not there.